skyportal/tests/external/test_moving_objects.py

```diff
@@ -0,0 +1,171 @@
+import os
+import time
+import uuid
+
+import arrow
+import pandas as pd
+import pytest
+from regions import Regions
+
+from skyportal.tests import api
+
+
+def add_telescope_and_instrument(api_class, super_admin_token, fields_ids=[]):
```

Comment:
I just wonder if this function belongs in a different file? One dedicated to adding and removing repeated uses.

Reply:
So I tried to put it under API but then it creates conflicts when importing the API method from __init__.py. It think that deserves it's own refactor where we put `api()` somewhere else first